example/miniapp/miniapp.cpp

```diff
@@ -49,17 +49,17 @@ int main(int argc, char** argv) {
     try {
 #ifdef ARB_MPI_ENABLED
         with_mpi guard(argc, argv, false);
-        context.distributed = mpi_context(MPI_COMM_WORLD);
+        context.distributed = distributed_context_handle(new distributed_context(mpi_context(MPI_COMM_WORLD)));
```

Comment:
Should this be `make_shared` ?

Reply:
Yes -- I think so. `distributed_context_handle` is just a `typedef` to `std::shared_ptr<distributed_context>`, so we should either directly call `std::make_shared<distributed_context>` or create a call that maps onto that.